### ISSUE TITLE
Use the rear cam

### ISSUE THREAD

**Issue by thorinas** [andz-bb/budibase-component-camera#3]
Hi, 

Would it be possible to allow the plugin to use the rear cam on a smart phone instead of the front cam?

All the best.

**Comment by dpitch** [comment]
yes please that would be awesome

**Comment by kodiak1988** [comment]
Yes, the abilty to switch between available cameras would be an awesome update! Plus the ability to record and upload videos would be great for my use case.

Thank you for your work so far 😀
